common.css

```diff
@@ -0,0 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Audiowide&display=swap");
+
+:root {
+  --blue-gradient: #40c9ff;
+  --violet-gradient: #e81cff;
+  font-size: 13px;
```

Comment:
I don't understand this font-size declaration here, and the use of px as well.

Reply:
the default root font-size is 16px which is changed to 13px as my component library was built on 13px root size.

Reply:
do ping me on discord f you still dont get it :)